core/spec/models/spree/calculator/default_tax_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+require 'spree/testing_support/calculator_helpers'
```

Comment:
I think a new folder would be clearest -- I didn't want to add one myself as I'm new to Solidus, so I just chose a place where I found other shared examples being defined currently.